sotodlib/toast/ops/load_context.py

```diff
@@ -87,6 +100,8 @@ class LoadContext(Operator):
 
     bands = List(list(), help="Only load this list of band values")
 
+    dets_select = Dict(dict(), help="The full detector selection dictionary to use")
```

Comment:
This seems a little cryptic to me. It is not a list of detectors but a dictionary. Is each key a name of a detector set?

Reply:
This dictionary is passed [directly to the Context method](https://github.com/simonsobs/sotodlib/blob/8c47e388f65841e4bc7b5af16b320100324970f3/sotodlib/core/context.py#L223).  I'll add a note in the trait help.

Reply:
Ok, trait help is reworded.